src/pages/Setup.js

```diff
@@ -171,12 +170,47 @@ function ChooseLiverySourcesPage({ data, setData }) {
   );
 }
 
-function SetupCompletePage() {
+/**
+ * Generates a section of the [ExpandableRow] from a field name and value.
+ *
+ * @param {Object} props
+ * @param {string} props.fieldName Section title
+ * @param {React.ReactNode} props.value Section value/description
+ *
+ * @return {React.ReactNode}
+ */
+function SetupCompleteSummary(props) {
```

Comment:
This component has been reused 3 times across files.

It'd be a great idea to refactor it, but what should it be called?